site/docs/master/install-overview.md

```diff
@@ -26,12 +26,7 @@ velero install \
 
 When using node-based IAM policies, `--secret-file` is not required, but `--no-secret` is required for confirmation.
 
-For provider-specific instructions, see:
-
-* [Run Velero on AWS][0]
-* [Run Velero on GCP][1]
-* [Run Velero on Azure][2]
-* [Use IBM Cloud Object Store as Velero's storage destination][4]
+For provider-specific instructions, see the navigation on the left under `Plugins`.
```

Comment:
Personally, I think it'd be easier for the user to navigate the install docs if we added *more* content to this Install Overview page about the compatible platforms and directly linked to instructions for each from here, rather than requiring them to jump into a whole different section on plugins to find links to install instructions for their platform. I think this page itself could use some reorg, maybe something like:

- Background/Prerequisites
    - Compatible object storage system, optional compatible block storage system(s), restic, ...
    - `velero install` or Helm chart
- Provider-specific instructions
    - Velero team-supported
    - Community
- `velero install` reference (details on all the different flags)

Thoughts?

Reply:
I made a proposed outline here based on feedback: https://hackmd.io/UpamNjb3SR2pSaJfeeHfzA

My thinking: let's nail down on this PR:
- the reorg as far as an outline
- move the existing content to fit this outline
- any writing/rewriting of content we can do in a new PR.

Reply:
sounds like a good plan to me! taking a look at the outline shortly

Reply:
The outline looks about right to me, I added a couple comments in the HackMD.

Reply:
Read your comments, make sense to me.